src/lib/cdk/tags.ts

```diff
@@ -0,0 +1,8 @@
+import {Construct} from "constructs";
+import {Tags} from "aws-cdk-lib";
+import {CommonStackProps} from "./stack";
+
+export function addCommonTags(construct: Construct, commonStackProps: CommonStackProps) {
+    Tags.of(construct).add("Owner", "Juho Friman");
+    Tags.of(construct).add("Dudate", "011219")
```

Comment:
I don't think we need these with our own dedicated AWS account?

Reply:
These were copied from the original config